src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const regularCost = 40;
+  const discountThreeDays = 20;
+  const discountSevenDays = 50;
+
+  if (days > 0 && days < 3) {
+    return days * regularCost;
+  }
+
+  if (days >= 3 && days < 7) {
+    return days * regularCost - discountThreeDays;
+  }
+
+  if (days >= 7) {
+    return days * regularCost - discountSevenDays;
+  }
```

Comment:
you repeated `days * regularCost` 3 times - move to variable

Reply:
also pass daysForDiscount into variables too

Reply:
please check